.github/workflows/ci.yml

```diff
@@ -250,9 +250,9 @@ jobs:
       - name: "Tar logs"
         if: always()
         run: |
-          cp -f doc/_build/latex/*.log ./logs-build-docs/
-          cp log.txt ./logs-build-docs/
-          tar cvzf ./logs-build-docs.tgz ./logs-build-docs
+          cp -f doc/_build/latex/*.log ./logs-build-docs/ || echo "Latex logs couldn't be found."

```

Comment:
Use "could not" here instead of using "couldn't" so it's the same as in the next line?